app/javascript/mastodon/features/ui/components/list_panel.tsx

```diff
@@ -31,6 +35,15 @@ export const ListPanel: React.FC = () => {
   const lists = useAppSelector((state) => getOrderedLists(state));
   const [expanded, setExpanded] = useState(false);
   const accessibilityId = useId();
+  const [ref, { height: viewHeight }] = useMeasure();
+  const { height, opacity } = useSpring({
+    from: { height: 0, opacity: 0 },
+    to: {
+      height: expanded ? viewHeight : 0,
+      opacity: expanded ? 1 : 0,
+    },
+    immediate: reduceMotion,
```

Comment:
i added a PR that sets this globally so we don't have to keep doing this: https://github.com/mastodon/mastodon/pull/35018